src/components/logservice/data.ts

```diff
@@ -0,0 +1,31 @@
+export type FormPropsData={

```

Comment:
React props should be defined in the component file (i.e., LogServiceButton.tsx)